rasa/nlu/selectors/response_selector.py

```diff
@@ -749,6 +773,14 @@ def _prepare_layers(self) -> None:
             (self.text_name, self.config),
             (self.label_name, label_config),
         ]:
+            if self.config.get(NUM_TRANSFORMER_LAYERS) == 0:
+                if SENTENCE not in self.data_signature[attribute]:
+                    raise ValueError(
```

Comment:
We avoid raise generic exceptions as much as possible. Over here it looks like this is a usage error. You could use `TFLayerConfigException` but that is specifically for TF layers. Here, it's about incorrect usage of the model. You can create a new exception class `TFModelConfigException` and use that.
Using these custom exceptions helps in sifting through exceptions caught inside sentry.